### PR TITLE
Log fixes

### PR DESCRIPTION
**Pull request by antmat**
Make logging control great again!

### CODE REVIEW

**Comment by antmat** [comment]
m?

**Comment by noxiouz** [comment]
LGTM

File: cocaine/tools/actions/logs.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def __init__(self, logging_service, filter_id):
 
     @gen.coroutine
     def execute(self):
-        filters = yield self.logging_service.remove_filter(self.filter_id)
+        filters = yield (yield self.logging_service.remove_filter(self.filter_id)).rx.get()
```
Comment (3Hren) [inline cocaine/tools/actions/logs.py]: Split as:
```python
channel = yield self.SERVICE.EVENT(ARGS...)
yield channel.rx.get()
```

File: cocaine/tools/actions/logs.py [hunk 2]
```diff
@@ -100,5 +100,5 @@ def __init__(self, logging_service, logger_name, filter_def, ttl):
 
     @gen.coroutine
     def execute(self):
-        filters = yield self.logging_service.set_cluster_filter(self.name, self.filter_def, self.ttl)
```
Comment (3Hren) [inline cocaine/tools/actions/logs.py]: Here too.

Comment (antmat) [inline cocaine/tools/actions/logs.py]: Fixed. This style was present in the whole file btw

**Comment by antmat** [comment]
@noxiouz merge, before @3Hren submits another PR!

**Pull request merged**